### PR TITLE
Docs: resonance testing fixup

### PR DESCRIPTION
**Pull request by brotherdust**
Added some clarifications around how to refer to adxl345 sensors when there are more than one.

### CODE REVIEW

**Comment by KevinOConnor** [comment]
@dmbutyugin - do you have any comments on this?

-Kevin

File: docs/Measuring_Resonances.md [hunk 1]
```diff
@@ -99,7 +99,12 @@ Measuring the resonances
 
 ## Checking the setup
 
-Now you can test a connection. In Octoprint, run `ACCELEROMETER_QUERY`. You
+Now you can test a connection.
+
+- For "non bed-slingers" (e.g. one accelerometer), in Octoprint, enter `ACCELEROMETER_QUERY`
+- For "bed-slingers" (e.g. more than one accelerometer), enter `ACCELEROMETER_QUERY CHIP=adxl345_x` where `adxl345_x` is the name of the chip as-entered (see: [bed-slinger](#bed-slinger-printers)).
+
+In Octoprint, run `ACCELEROMETER_QUERY`. You
```
Comment (dmbutyugin) [inline docs/Measuring_Resonances.md:107]: I think the first sentence is redundant now. I think we can simply leave here

You should see the current measurements from the accelerometer,
including the free-fall acceleration, e.g.

**Comment by dmbutyugin** [comment]
@KevinOConnor yes, that's a bug in the documentation. The PR looks good overall.

@brotherdust Thanks! I was also thinking about maybe renaming the accelerometers in the two accelerometers example to
```
[adxl345 hotend]
# Assuming adxl345 on the hotend is connected to an RPi
cs_pin: rpi:None

[adxl345 bed]
# Assuming adxl345 on the bed is connected to a printer MCU board
cs_pin: ...  # Printer board SPI chip select (CS) pin

[resonance_tester]
accel_chip_x: adxl345 hotend
accel_chip_y: adxl345 bed
probe_points: ...
```
and with appropriate references. However, I'm not 100% sure which version is more readable and less confusing. In principle, we can leave it as-is.

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**